cluster/gce/util.sh

```diff
@@ -1837,6 +1843,7 @@ function update-or-verify-gcloud() {
     sudo_prefix="sudo"
   fi
   # update and install components as needed
+  # shellcheck disable=SC2086
```

Comment:
why?

Reply:
If we were to quote `gcloud_prompt`, say it contained _foo bar_, we would have the single argument `foo bar` instead of the two arguments `foo` `bar` passed to gcloud. And if gcloud_prompt was empty, we would get an empty argument instead of no argument.